packages/core/package.json

```diff
@@ -17,19 +17,22 @@
       "types": "./dist/lib/message/version_0.d.ts",
       "import": "./dist/lib/message/version_0.js"
     },
-    "./lib/message/topic_only_message": {
-      "types": "./dist/lib/message/topic_only_message.d.ts",
-      "import": "./dist/lib/message/topic_only_message.js"
-    },
     "./lib/base_protocol": {
       "types": "./dist/lib/base_protocol.d.ts",
       "import": "./dist/lib/base_protocol.js"
+    },
+    "./constants": {
+      "types": "./dist/constants/index.d.ts",
+      "import": "./dist/constants/index.js"
     }
   },
   "typesVersions": {
     "*": {
       "lib/*": [
         "dist/lib/*"
+      ],
+      "constants/*": [
```

Comment:
Any reason why is `constants` out of the `lib` dir? 

I said move it in the `lib` dir. We can remove the `lib` dir if it bothers you.